### PR TITLE
Add English translation for component message

### PR DESCRIPTION
**Pull request by u9lyfish**
Add English translation for component message

### CODE REVIEW

**Comment by benjycui** [comment]
@manV I am not sure if you have time to review this.

File: components/message/demo/duration.md [hunk 1]
```diff
@@ -1,17 +1,25 @@
 ---
 order: 2
-title: 修改延时
+title:
+  zh-CN: 修改延时
+  en-US: Modify display duration
```
Comment (afc163) [inline components/message/demo/duration.md]: Customize duration

File: components/message/demo/duration.md [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
 ---
 
+## zh-CN
+
 自定义时长 `10s`，默认时长为 `1.5s`。
 
+## en-US
+
+Customize message display duration to `10s`, while default duration is `1.5s`。
```
Comment (afc163) [inline components/message/demo/duration.md]: Customize message display duration from default `1.5s` to `10s`

`。` => `.`

**Comment by benjycui** [comment]
@u9lyfish rebase 一下？

**Comment by benjycui** [comment]
https://github.com/ant-design/ant-design/commit/0869e2959aea2a2d5bf3f8e2e227963122da07a8

**Pull request closed without merging**